frontend/app/model/Benefits.scala

```diff
@@ -28,6 +28,8 @@ object Benefits {
     BenefitItem("email_updates", "Regular events email"),
     BenefitItem("offers_competitions", "Offers and competitions"),
     BenefitItem("priority_booking", "48hrs priority booking"),
+    BenefitItem("books_or_tickets", "6 tickets or 4 books", isNew = true),
```

Comment:
If possible I would avoid hardcoding the number 6, and instead use `Config.zuoraFreeEventTicketsAllowance`